src/colab_noosfero/data_importer.py

```diff
@@ -100,7 +100,7 @@ def fetch_communities(self):
         json_data = self.get_json_data(url, 1, timestamp=timestamp,
                                        order="updated_at ASC")
 
-        if len(json_data) == 0:
+        if not len(json_data) or not len(json_data['communities']):
```

Comment:
This will fail if *len(json_data) > 1* and json_data dict has no key *'communities'*. The same will happen in the lines 148 and 170.